packages/testlab/src/test-sandbox.ts

```diff
@@ -37,11 +38,16 @@ export class TestSandbox {
    * Will create a directory if it doesn't already exist. If it exists, you
    * still get an instance of the TestSandbox.
    *
-   * @param path - Path of the TestSandbox. If relative (it will be resolved relative to cwd()).
+   * @param rootPath - Path of the TestSandbox. If relative it will be resolved
+   * against the current directory.
+   * @param subdir - Subdirectory for the sandbox. If not provided, the sandbox
+   * will automatically generate a uuid as the name of the subdirectory.
    */
-  constructor(path: string) {
+  constructor(rootPath: string, subdir?: string) {
+    // Create a unique subdirectory so that sandboxes can be used in parallel
+    subdir = subdir ?? v1();
```

Comment:
I have two comments:

1. I believe the `v1` algorithm is considered as less reliable than `v4`. UUID generation is being discussed as a new JavaScript features and they proposal is to use `v4` algorithm only (or at least as the default). Are there any reasons for preferring v1 over v4 for our sandbox?

2. Do we actually need full UUID semantics? If we want to achieve isolation between (child) processes, then `process.pid` is a unique value that's much shorter than a uuid and does not require any new dependencies.

Reply:
I'll switch to `v4`.

If mocha decides to use worker threads for parallel testing, process.pid won't be unique.

Reply:
I switched to https://nodejs.org/api/fs.html#fs_fs_mkdtempsync_prefix_options and removed uuid dependency.

Reply:
👏 I love that we can avoid adding a new dependency and create short subdirectory names - this will make it easier to inspect sandbox content when troubleshooting tests.